http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/StandardEnabledUserDirectives.scala

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.dsde.workbench.leonardo.http
+package api
+
+import akka.http.scaladsl.server.Directive
+// TODO akka require isn't in this package - where is it? see https://doc.akka.io/docs/akka-http/current/routing-dsl/directives/custom-directives.html#require-trequire
```

Comment:
And also, how does it work - I clearly don't understand the syntax needed to define and use Directives